pkg/streamer/streamer.go

```diff
@@ -32,12 +32,25 @@ type Streamer struct {
 type Config struct {
 	Parsers              []parser.PublicationParser // Parsers used to open a publication, in addition to the default parsers.
 	IgnoreDefaultParsers bool                       // When true, only parsers provided in parsers will be used.
-	InferA11yMetadata    bool                       // When true, additional accessibility metadata will be infered from the manifest.
+	InferA11yMetadata    InferA11yMetadata          // When set, additional accessibility metadata will be infered from the manifest.
 	InferPageCount       bool                       // When true, will infer `Metadata.NumberOfPages` from the generated position list.
 	ArchiveFactory       archive.ArchiveFactory     // Opens an archive (e.g. ZIP, RAR), optionally protected by credentials.
 	HttpClient           *http.Client               // Service performing HTTP requests.
 }
 
+type InferA11yMetadata string
```

Comment:
Why a string instead of e.g. uint8 with values 1,2,3 (0 is unset)? It would be more efficient, and since there are const values defining the options, it's not very different for implementers. The only different is needing to translate from command line string to the option

Reply:
Good idea. I started from the CLI side and moved this to the Streamer without much thought. The translation from strings is already done anyways.